lib/membrane_file/source.ex

```diff
@@ -17,17 +17,27 @@ defmodule Membrane.File.Source do
                 spec: pos_integer(),
                 default: 2048,
                 description: "Size of chunks being read"
+              ],
+              content_format: [
+                spec: module() | nil,
+                default: nil,
+                description: "Caps which will be sent from Source"
```

Comment:
This description seems to be taken from the old version of the PR :wink: 
I'd rephrase it to better reflect what will happen with that content_format